### ISSUE TITLE
Search Under Latency Constraints, How to interpret results?

### ISSUE THREAD

**Issue by soyebn** [Alibaba-MIIL/HardCoReNAS#2]
Very impressive improvement in speed over other SOTA. Thanks for sharing the code.

I ran the following command to search 25 msec latency model for P100 GPU. 
CUDA_VISIBLE_DEVICES=0 python -u ./search.py path_to_imagenet --train_percent=80 --bcfw_steps=10000 --initial-checkpoint=./pretrained/w_star_076b2ed3.pth --inference_time_limit=25 --lut_filename=./latency/LUT_GPU.pkl --batch-size 6 --amp 0

I am pointing to pre-trained model at, https://miil-public-eu.oss-eu-central-1.aliyuncs.com/public/HardCoReNAS/w_star_076b2ed3.pth

I got the following results,
**Test child model: [8333/8333]  Time: 0.821 (0.026)  Loss:  3.6528 (0.9987)  Acc@1: 50.0000 (76.4600)  Acc@5: 50.0000 (93.1140)
Latency_predicted=0.07242456756666653, latency_measured=0.26521933794021607, diff=0.19279477037354953**

How do I interpret this result? Did I get 72 msec model with 76.46 Top1 accuracy. I was expecting latency close to 25 msec.
